addon-test-support/helpers.js

```diff
@@ -13,6 +13,11 @@ import Ember from 'ember';
  */
 export default function isQueueDrained(app, queueName) {
   const schedulerService = app.__container__.lookup('service:scheduler');
-  const queue = schedulerService.queues[queueName];
+  const queues = schedulerService.queues;
+  if (!queues) {
```

Comment:
This is to specifically ensure that we return `true` if the service has already started destruction?

Reply:
yup! That is correct